AnimationEncoder/doxy.cfg

```diff
@@ -1,4 +1,4 @@
-# Doxyfile 1.8.9.1
+# Doxyfile 1.9.1
```

Comment:
Linux Mint 20 has version 1.8.17 .

As we don't need bleeding edge doxygen stuff, I'd say stay compatible with whatever current LTS Ubuntu has (Mint being a derivative of Ubuntu, it's likely Ubuntu doesn't use `1.9` either.)